frontend/src/services/gitFunctions.js

```diff
@@ -0,0 +1,22 @@
+var Git = require("nodegit");
```

Comment:
Do not use var, there are some inconsistencies caused by it, also JavaScript doesn't start the variables name with uppercase